frontend/www/js/omegaup/user/profile.ts

```diff
@@ -0,0 +1,173 @@
+                rotation: -45,
+              },
+            },
+            yAxis: {
+              min: 0,
+              title: { text: T.profileStatisticsNumberOfSolvedProblems },
+              stackLabels: {
+                enabled: false,
+                style: {
+                  fontWeight: 'bold',
+                  color: 'gray',
+                },
+              },
+            },
+            legend: {
+              align: 'right',
+              x: -30,
+              verticalAlign: 'top',
+              y: 25,
+              floating: true,
+              backgroundColor: 'white',
+              borderColor: '#CCC',
+              borderWidth: 1,
+              shadow: false,
+            },
+            tooltip: {
+              headerFormat: '<b>{point.x}</b><br/>',
+              pointFormat:
+                '{series.name}: {point.y}<br/>Total: {point.stackTotal}',
+            },
+            plotOptions: {
+              column: {
+                stacking: 'normal',
+                dataLabels: {
+                  enabled: false,
+                  color: 'white',
+                },
+              },
+            },
+            series: [],
+          },
+          aggregateStatisticOptions: {
+            title: {
+              text: ui.formatString(T.profileStatisticsVerdictsOf, {
+                user: payload.profile?.username,
+              }),
+            },
+            chart: {
+              plotBackgroundColor: null,
+              plotBorderWidth: null,
+              plotShadow: false,
+              type: 'pie',
+            },
+            xAxis: {
+              title: { text: '' },
+            },
+            yAxis: {
+              title: { text: '' },
+            },
+            tooltip: { pointFormat: '{series.name}: {point.y}' },
+            plotOptions: {
+              pie: {
+                allowPointSelect: true,
+                cursor: 'pointer',
+                dataLabels: {
+                  enabled: true,
+                  color: '#000000',
+                  connectorColor: '#000000',
+                  format:
+                    '<b>{point.name}</b>: {point.percentage:.1f} % ({point.y})',
+                },
+              },
+            },
+            series: [
+              {
+                name: T.profileStatisticsRuns,
+                data: [],
+              },
+            ],
+          },
+        },
+        on: {
+          'update-period-statistics': (e, categories, data) => {
+            // console.log(e);
+            // console.log(categories);
+            // console.log(data);
+            e.periodStatisticOptions.xAxis.categories = categories;
+            e.periodStatisticOptions.series = data;
+          },
+          'update-aggregate-statistics': (e) =>
+            (e.aggregateStatisticOptions.series[0].data =
+              e.normalizedRunCounts),
+        },
+      });
+    },
+
+    computed: {
+      rank: function () {
```

Comment:
```suggestion
      rank: () => {
```

también, no sería mejor mandarle `payload.profile?.classname` al componente y que él se encargue de convertirlo?